src/main/java/Lumiere.java

```diff
@@ -0,0 +1,163 @@
+import java.util.*;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class Lumiere {
+    private static String fileLocation = "./lumiere.txt";
+    private static Path filePath = Paths.get(fileLocation);
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+
+        List<Task> list = loadTasksFromFile();
+        System.out.println("Hello! I'm Lumiere\n" + "Tell me what I can do for you, be my guest!\n");
+
+        while (true) {
+            String command = sc.nextLine();
+            if (command.equals("list")) {
+                System.out.println("Here are the tasks in your list:");
+                for (int i = 0; i < list.size(); i++) {
+                    System.out.println(Integer.toString(i + 1) + "." + list.get(i).stringify());
```

Comment:
As mentioned above, if you override `toString()` instead of using `stringify()` you should be able to print your tasks like this.
```suggestion
                    System.out.println(Integer.toString(i + 1) + "." + list.get(i));
```